.github/workflows/pr-test.yml

```diff
@@ -39,32 +39,12 @@ jobs:
         run: |
           yarn --frozen-lockfile
 
-      - name: Config git core.quotePath
+      - name: Get changed documents
         run: |
-          # If you don't do this, the get-diff-action won't be able to
-          # notice files that contain non-ascii characters.
-          # I.e.
-          #
-          #    ▶ git status
-          #    Changes not staged for commit:
-          #    ...
-          #   modified:   "files/en-us/glossary/b\303\251zier_curve/index.html"
-          #
-          # But after you set `core.quotePath` you get:
-          #
-          #    ▶ git status
-          #    Changes not staged for commit:
-          #    ...
-          #   modified:   "files/en-us/glossary/bézier_curve/index.html"
-          #
-          # Now, this gets used by the `git diff ...` inside get-diff-action.
-          git config --global core.quotePath false
```

Comment:
not necessary for `gh` cli:

![image](https://user-images.githubusercontent.com/15844309/193401331-68138f8e-ff4c-4f7d-9bc6-651a177b9f26.png)